### ISSUE TITLE
Unable to delete unwanted device

### ISSUE THREAD

**Issue by didierga** [andyholmes/gnome-shell-extension-gsconnect#64]
I run Fedora with gnome-shell and I have VirtualBox installed. 
I booted Fedora KDE in a virtual machine.
Since this boot and as KDE include KDE Connect, I have my KDE user listed as GSConnect device and in the gnome-shell top bar a red icon who indicates this KDE user is not associated with GSConnect.
I did not find any way to delete this unwanted device and by the way suppress the red icon in gnome-shell top bar.
How can I do that ?

**Comment by andyholmes** [comment]
Hi, you should be able to hide the icon using the setting **Show Unpaired** described here:

https://github.com/andyholmes/gnome-shell-extension-gsconnect/wiki/Preferences#show-unpaired

Other than that, unpaired devices should be automatically removed like in KDE Connect, when they are no longer connected.

**Comment by andyholmes** [comment]
Yeah, I tried once or twice communicating between GSConnect and KDE Connect in a virtual machine, but although most things worked as expected, it got confused a fair amount too. It's not something I'd formally support at this point.

As far removing devices, I believe that GSConnect should do it if a device is unpaired *when* it disconnects, and I think it also prunes unpaired devices when the preferences window is closed. However, it never actually erases the information from GSettings, more just removes it from a list of devices to automatically try to connect to. There was I reason I did that, I just can't remember it anymore.

Does hiding unpaired devices solve the issue, well enough?

**Comment by z3bu** [comment]
Hello, thanks for this amazing extension, I just leave this message so it could help others.

I switched from mconnect to gsconnect, my mconnect laptop "device profile" was listed as a device on gsconnect itself and was able to be paired (the laptop paired itself), I have been able to clean gsconnect list  by running `kdeconnect-cli -l` .

**Comment by didierga** [comment]
Andy, as you suggested, I have been able to manually hide unpaired icon switching off "Show unpaired" in settings. Thanks for your help.
